File: introduction/ITP1_10_B-Triangle.cpp
#include <iostream>
#include <cmath>
#include <iomanip>
using namespace std;
int main() {
  double a, b, c, h;
  cin >> a >> b >> c;
  c = c*3.1415926/180;
  h = b*sin(c);
  cout << fixed << setprecision(5) << a*h*.5 << endl << a+b+sqrt(a*a+b*b-2*a*b*cos(c)) << endl << h << endl;
  return 0;
}

File: introduction/ITP1_10_B-Triangle.java
import java.util.*;
public class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    double a = sc.nextDouble(), b = sc.nextDouble(), c = sc.nextDouble() * Math.PI/180, h = b*Math.sin(c);
    System.out.printf("%f %f %f", a*h*.5, a+b+Math.sqrt(a*a+b*b-2*a*b*(Math.cos(c))), h);
  }
}
